auth/cloud-client/snippets_test.py

```diff
@@ -31,3 +32,12 @@ def test_explicit():
 
     with mock.patch('io.open', open_mock):
         snippets.explicit()
+
+
+def test_explicit_compute_engine():
+    adc, project = google.auth.default()
+    credentials_patch = mock.patch(
```

Comment:
Why assign this to a variable and then use it as a context manager instead of inlining it with the "with" as in the test above?